iml-system-rpm-tests/tests/ldiskfs_test.rs

```diff
@@ -15,7 +14,7 @@ async fn run_test(config: &vagrant::ClusterConfig) -> Result<(), SystemTestError
         }),
         vec![("base_monitored".into(), &config.storage_servers()[..])]
             .into_iter()
-            .collect::<HashMap<String, &[&str]>>(),
+            .collect(),
```

Comment:
This doesn't seem to serve a purpose anymore.

Reply:
ah right, it's already a vector. I'll update.